test/validation/api/ipsec/ipsec_test_in.c

```diff
@@ -1597,6 +1597,42 @@ static void test_in_ipv6_esp_udp_null_sha256_lookup(void)
 	ipsec_sa_destroy(sa);
 }
 
+static void test_ipsec_print(void)
+{
+	odp_ipsec_print();
+}
+
+static void test_ipsec_sa_print(void)
+{
+	odp_ipsec_tunnel_param_t in_tunnel;
+	odp_ipsec_sa_param_t param_in;
+	odp_ipsec_capability_t capa;
+	uint32_t window_sz = 32;
+	odp_ipsec_sa_t in_sa;
+
+	CU_ASSERT(odp_ipsec_capability(&capa) == 0);
+
+	memset(&in_tunnel, 0, sizeof(odp_ipsec_tunnel_param_t));
+
+	odp_ipsec_print();
+
+	ipsec_sa_param_fill(&param_in,
+			    true, false, 0x1234, &in_tunnel,
+			    ODP_CIPHER_ALG_AES_GCM, &key_mcgrew_gcm_3,
+			    ODP_AUTH_ALG_AES_GCM, NULL,
```

Comment:
A commonly supported algorithm should be selected, so that this test case will likely run. Would e.g. ODP_CIPHER_ALG_AES_CBC be more likely supported than GCM (not sure if there's a difference)?

Reply:
Every IPsec implementation must support aes-cbc-128 and hmac-sha1-96, so using those might be slightly better, although aes-gcm is very common too.

Reply:
In V4, I have updated SA print test function to use aes-cbc-128 cipher and sha1 auth.

Also I have cleaned up the test function.